cloudprober/templates/configmap.yaml

```diff
@@ -4,7 +4,7 @@ kind: ConfigMap
 metadata:
   name: {{ include "cloudprober.fullname" . }}
 data:
-  cloudprober.{{- .Values.configFormat }}: |
+  {{- .Values.configFileName | default "cloudprober" }}.{{- .Values.configFormat }}: |
```

Comment:
Just a couple of more things:
- Instead of setting the default here, it's better to just put in values.yaml, like how we do for the config format.
- Please add this variable to values.yaml for documentation and for default value, likely here:
https://github.com/cloudprober/helm-charts/blob/56e1e69585fbf519c0fd6bee9fcbb1468fbaabcb/cloudprober/values.yaml#L23